cl/opinion_page/views.py

```diff
@@ -642,10 +646,25 @@ def reporter_or_volume_handler(
                 "reporter": reporter,
                 "volume_names": volume_names,
                 "volume": volume,
-                "private": True,
+                "private": False,
             },
         )
 
+    volume_int = int(volume)
+    volumes = list(
+        (
+            OpinionCluster.objects.filter(
+                citations__reporter=reporter,
+            )
+            .annotate(as_integer=Cast("citations__volume", IntegerField()))
+            .filter(as_integer__range=[volume_int - 1, volume_int + 1])
+            .values_list("as_integer", flat=True)
+            .distinct()
+        )
+    )
```

Comment:
Would this work as a query directly on the Citation table instead of doing a join?

Reply:
👍  yes it would